youtube_dl/extractor/peertube.py

```diff
@@ -513,10 +537,25 @@ def _real_extract(self, url):
             formats.append(f)
         self._sort_formats(formats)
 
-        def account_data(field):
-            return try_get(video, lambda x: x['account'][field], compat_str)
+        video_description = self._download_json(
+            'https://%s/api/v1/videos/%s/description' % (host, video_id), video_id, fatal=False)
+
+        description = None
+        if isinstance(video_description, dict):
+            description = video_description.get('description')
+
+        subtitles = self.extract_subtitles(host, video_id)
+
+        def try_get_second_level_data(section, field, type_):
```

Comment:
`try_get_second_level_data` -> `data`

Reply:
As in, rename the function? Alright.
Would this cause any readability issues for future contributors as it is a special case (only goes down two levels)?